packages/action-button/test/action-button.test.ts

```diff
@@ -343,5 +317,41 @@ describe('ActionButton', () => {
                 },
             });
         });
+
+        it('warns that `variant` is deprecated', async () => {
+            const el = await fixture<ActionButton>(html`
+                <sp-action-button static="white">Button</sp-action-button>
+            `);
+
+            await elementUpdated(el);
+
+            expect(consoleWarnStub.called).to.be.true;
+            const spyCall = consoleWarnStub.getCall(0);
+            expect(
+                (spyCall.args.at(0) as string).includes('"static"'),
+                'confirm static-centric message'
+            ).to.be.true;
+            expect(spyCall.args.at(-1), 'confirm `data` shape').to.deep.equal({
+                data: {
+                    localName: 'sp-action-button',
+                    type: 'api',
+                    level: 'default',
```

Comment:
@rubencarvalho Should level be 'default'?

Reply:
Oops, nice catch.